### PR TITLE
[Canvas V2][3/n] Implement React event row

### PR DESCRIPTION
**Pull request by taneliang**
Stacked on #32. Part of #30.

## Test Plan

Default:

image

Zooming:

image

Vertical panning:

image

### CODE REVIEW

File: src/canvas/renderCanvas.js [hunk 1]
```diff
@@ -76,18 +79,126 @@ import {REACT_TOTAL_NUM_LANES} from '../constants';
...
   context.fillRect(0, 0, canvasWidth, canvasHeight);
 }
 
-/**
- * Render all charting data (once it's loaded and processed) within the
- * "scrollable" region.
- */
-// TODO (windowing) We can avoid rendering all of this if we've scrolled some of it off screen.
-function renderReactMarksAndMeasures(
+function renderReactEventRow(
```
Comment (jevakallio) [inline src/canvas/renderCanvas.js:100]: Heading comments describing the purpose of the function would be nice here

File: src/canvas/renderCanvas.js [hunk 2]
```diff
@@ -76,18 +79,126 @@ import {REACT_TOTAL_NUM_LANES} from '../constants';
...
+    const y = baseY + REACT_EVENT_SIZE / 2 - panOffsetY;
+
+    context.beginPath();
+    context.fillStyle = fillStyle;
+    context.arc(x, y, circumference / 2, 0, 2 * Math.PI);
+    context.fill();
+  }
+}
+
+function renderReactMeasures(
```
Comment (jevakallio) [inline src/canvas/renderCanvas.js:225]: Comment block explaining what measures are would help here

Comment (jevakallio) [reply src/canvas/renderCanvas.js:225]: This should also indicate `: number` for return type

**Review by jevakallio — APPROVED** [review]
LGTM with a couple of nits 👍

File: src/canvas/renderCanvas.js (hunk 2 above)
Comment (taneliang) [reply src/canvas/renderCanvas.js:225]: 👍 I'll add JSDoc comments to the above functions in this PR, and I've added `: number` in the 1/n PR

File: src/canvas/renderCanvas.js (hunk 1 above)
Comment (taneliang) [reply src/canvas/renderCanvas.js:100]: Added!

**Comment by taneliang** [comment]
Rebase and add JSDoc comments to some render* functions

**Comment by taneliang** [comment]
Rebased on master

**Pull request merged**